### PR TITLE
Add: Initial setup for configuring apps 

### PR DESCRIPTION
**Pull request by zewolfe**
### What does this PR do?

Add scripts for configuring apps and `gcp-compute-persistent-disk-csi-driver-app`
Towards https://github.com/giantswarm/roadmap/issues/1453

- [x] CHANGELOG.md has been updated (if it exists)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## [Unreleased]
+- Add script to configure an apps workload identity
```
Comment (fiunchinho) [inline CHANGELOG.md]: I don't think I understand , maybe it's better this way?

```suggestion
- Add script to configure apps using workload identity
```

File: Makefile.custom.mk [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+endif
+
+.PHONY: ensure-gcp-envs
+ensure-gcp-envs:
+ifndef GCP_PROJECT_ID
+	$(error GCP_PROJECT_ID is undefined)
+endif
+
+.PHONY: all
+all: capg-firewall-rule-operator capg-operator cluster-control-plane-node cluster-worker-node csi-driver-app dns-operator-gcp external-dns-app fleet-membership-operator-gcp
```
Comment (fiunchinho) [inline Makefile.custom.mk]: Sorry, my OCD kicked in: why `capg-operator` instead of `cluster-api-provider-gcp-app`?

File: README.md [hunk 3]
```diff
@@ -1,30 +1,4 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/template.svg?style=shield)](https://circleci.com/gh/giantswarm/template)
+# workload-identity-gcp-utils
 
-# REPOSITORY_NAME
+A collection of scripts to help you configure and use workload identity on GCP.
```
Comment (fiunchinho) [inline README.md:3]: I'd add a few words saying that just running `make` would create all the service accounts required by GS.

**Review by fiunchinho — CHANGES_REQUESTED** [review]
Few suggestions, looks good otherwise

File: Makefile.custom.mk (hunk 2 above)
Comment (mnitchev) [reply Makefile.custom.mk]: done

File: README.md (hunk 3 above)
Comment (mnitchev) [reply README.md:3]: Done, also added `make help` entries

**Review by fiunchinho — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,30 +1,29 @@
...
+
+### Prerequisites 
+
+1. Install [gcloud cli](https://cloud.google.com/sdk/docs/install)
+1. Run `gcloud auth login`
+1. Make sure you log in with a user that has the permissions to create service
+   accounts, roles and assing roles to service accounts on the project level as
+   well as assign service account users.
+
+### Installing everyting
```
Comment (fiunchinho) [inline README.md]:
```suggestion
### Installing everything
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## [Unreleased]
+- Add script to configure apps using workload identity
+- Add script to configure `gcp-compute-persistent-disk-csi-driver-app`
```
Comment (fiunchinho) [inline CHANGELOG.md]: Why that app has its own entry in the changelog? isn't it included on the first item?

Comment (mnitchev) [reply CHANGELOG.md]: It used to be what this PR is about, but we just added everything else

**Pull request merged**